src/main/java/org/kohsuke/github/GHPullRequestSearchBuilder.java

```diff
@@ -0,0 +1,490 @@
+        return this;
+    }
+
+    /**
+     * Updated gh pull request search builder.
+     *
+     * @param from
+     *            the updated starting from
+     * @param to
+     *            the updated ending to
+     * @return the gh pull request search builder
+     */
+    public GHPullRequestSearchBuilder updated(LocalDate from, LocalDate to) {
+        String updatedRange = from.format(DateTimeFormatter.ISO_DATE) + ".." + to.format(DateTimeFormatter.ISO_DATE);
+        q("updated", updatedRange);
+        return this;
+    }
+
+    /**
+     * Label gh pull request search builder.
+     *
+     * @param label
+     *            the label
+     * @return the gh pull request search builder
+     */
+    public GHPullRequestSearchBuilder label(String label) {
+        q("label", label);
+        return this;
+    }
+
+    /**
+     * Labels gh pull request search builder.
+     *
+     * @param labels
+     *            the labels
+     * @return the gh pull request search builder
+     */
+    public GHPullRequestSearchBuilder inLabels(Iterable<String> labels) {
+        q("label", String.join(",", labels));
+        return this;
+    }
+
+    /**
+     * Title like search term
+     *
+     * @param title
+     *            the title to be matched
+     * @return the gh pull request search builder
+     */
+    public GHPullRequestSearchBuilder titleLike(String title) {
+        q(title + " in:title");
+        return this;
+    }
+
+    /**
+     * Order gh pull request search builder.
+     *
+     * @param direction
+     *            the direction
+     * @return the gh pull request search builder
+     */
+    public GHPullRequestSearchBuilder order(GHDirection direction) {
+        req.with("order", direction);
+        return this;
+    }
+
+    /**
+     * Sort gh pull request search builder.
+     *
+     * @param sort
+     *            the sort
+     * @return the gh pull request search builder
+     */
+    public GHPullRequestSearchBuilder sort(GHPullRequestSearchBuilder.Sort sort) {
+        req.with("sort", sort);
+        return this;
+    }
+
+    @Override
+    public GHPullRequestSearchBuilder q(String term) {
+        super.q(term);
+        return this;
+    }
+
+    @Override
+    public PagedSearchIterable<GHPullRequest> list() {
+        this.q("is:pr");
+        return super.list();
+    }
+
+    @Override
+    protected String getApiUrl() {
+        return "/search/issues";
+    }
+
+    /**
+     * The sort order values.
+     */
+    public enum Sort {
+
+        /** The comments. */
+        COMMENTS,
+        /** The created. */
+        CREATED,
+        /** The updated. */
+        UPDATED,
+        /** The relevance. */
+        RELEVANCE
+
+    }
+
+    static GHPullRequestSearchBuilder from(GHPullRequestSearchBuilder searchBuilder) {
+        GHPullRequestSearchBuilder builder = new GHPullRequestSearchBuilder(searchBuilder.root());
```

Comment:
With the way this is used, you could have someone calling a GHRepository search method, but no using that objects `GitHub` connection.  

```suggestion
    static GHPullRequestSearchBuilder from(GitHub root, GHPullRequestSearchBuilder searchBuilder) {
        GHPullRequestSearchBuilder builder = new GHPullRequestSearchBuilder(root);
```